Dockerfile

```diff
@@ -1,11 +1,6 @@
 # Dockerfile
-# Pierre-André Noël, May 12th 2020
-# Copyright © Element AI Inc. All rights reserved.
```

Comment:
Is this deletion mandatory for you to use this? If yes, then I can open a ticket with my org's Legal to see what I can do... But if not, could you please but these lines back? It would be much simpler...

Reply:
I know nothing about legal matters but this file has had subsequent additions since your original draft, both from me and @daanzu.

For this to be useful in general I would like it to be distributed along with the rest of Pynini, for which the copyright is held by Google and which is released under an Apache 2.0 license. I don't think I should distribute a file with some other org's copyright declaration without a full legal audit.

Is there any reason we need copyright protection here in the first place? Maybe we could just list authors.

Reply:
I am not a legal person either, but my personal understanding is that for something to be licensed under Apache 2.0, there must be some copyright owner(s). I would be fine with adding your names as contributors, but the decision of "gifting" the file to Google is above my pay grade. I'll open a Legal ticket to inquire and will keep you posted here.

Reply:
Thanks, yes. If the answer is yes then I'd set up a contributor agreement and have you "contribute" it to Pynini itself. (https://cla.developers.google.com/about/google-individual). If the answer is no I can ask on my side. Either way we can definitely get this figured out.

Reply:
So I discussed with the appropriate folks at ServiceNow (whom acquired Element AI in January 2021).

Their proposed solution is to keep the copyright notice and associated lines, and someone VP level at ServiceNow would sign the Google contributor agreement you linked.

Please let me know if this would be an acceptable resolution.

Reply:
Unfortunately, I don't think that solves any of the issues, since the CLA grants the copyright to Google, so that would seemingly contradict the copyright notice. I'm going to have to loop Google legal into this, I think, sorry to say.

Reply:
Answer from appropriate folks at ServiceNow:

> The CLA grants a copyright license to Google.  That is paragraph 2.

Reply:
I guess I don't understand the difference between "Copyright " and "Copyright License"---and I'd probably have to get my JD to get it ;) Anyways I'll work on this tomorrow with legal.